ovos_tts_plugin_server/__init__.py

```diff
@@ -28,8 +28,18 @@ def __init__(self, *args, **kwargs):
 
     @property
     def host(self) -> Optional[str]:
+        self.log.warning("Deprecation Warning: use self.urls instead")
```

Comment:
`ovos_utils.log.log_deprecation` can be used here to use consistent formatting to find in logs (or the `ovos_utils.log.deprecated` decorator could be used to do the same)